src/components/LoanCards/LoanTags/LoanTagV2.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+	<div v-if="!!variation && timeLeftMs > 0" class="tw-text-small tw-font-medium tw-mt-0.5" style="color: #CE4A00;">
+		{{ tagText }}
+		<countdown-timer
+			v-if="variation === 'ending-soon'"
+			:time="timeLeftMs"
+		/>
+	</div>
+</template>
+
+<script>
+import { differenceInDays, parseISO } from 'date-fns';
+import CountdownTimer from '@/components/BorrowerProfile/CountdownTimer';
+
+export default {
+	name: 'LoanTagV2',
```

Comment:
Please add a follow-up ticket for if/when the experiment is successful to write unit tests. This would be a great component to get some jest tests written for, since it has an isolated sentence output

Reply:
👍  https://kiva.atlassian.net/browse/CORE-1053 donezo!